packages/transport/src/modules/tokens/AnonymousTokenController.ts

```diff
@@ -19,10 +19,14 @@ export class AnonymousTokenController {
     }
 
     public async loadPeerTokenByReference(tokenReference: TokenReference): Promise<Token> {
+        // TODO: add the token reference forIdentity once available
```

Comment:
TODO in code - remove please. ALso we don't have to change sth here when the updated truncated refereces are landing as the only thing that changes will be the length of the string.